src/hooks/useUser/useUser.tsx

```diff
@@ -30,9 +33,12 @@ ActiveUserContext.displayName = 'ActiveUserContext'
 
 export const ActiveUserProvider: React.FC = ({ children }) => {
   const { activeUserState, setActiveUser, resetActiveUser } = useActiveUserStore()
+  const [openDialog, closeDialog] = useDialog()
+  const browser = useCheckBrowser()
```

Comment:
Let's use the new browser checks from https://github.com/Joystream/atlas/pull/710

I think we should get rid of `useCheckBrowser`